onnxscript/optimizer/_constant_folding.py

```diff
@@ -77,15 +77,15 @@ class Replacement:
 
 class OptimizerState:
     def __init__(self):
-        self._sym_value_map: dict[ir.Value, Any] = {}
+        pass
```

Comment:
Even the changes here can be undone if ir.Value is usable as dictionary key. That is preferable to use the `.meta` option ... because it makes it easy to discard temporary dictionaries once they are no longer needed ... info stored in `.meta` will persist unless we have some mechanism to explicitly free it.